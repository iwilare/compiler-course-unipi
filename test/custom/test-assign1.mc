int main() {
    int a;

    a = 0;

    a += 10;
    print(a);

    a += a + 10;
    print(a);

    a -= 10;
    print(a);

    a %= 3;
    print(a);

    a *= 2;
    print(a);

    a /= 2;
    print(a);

    return 0;
}
