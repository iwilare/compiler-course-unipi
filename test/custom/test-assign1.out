10
30
20
2
4
2
